### DISCUSSION TITLE
Dashboard where we can get all CICD and PRs details

### DISCUSSION THREAD

**Discussion by conrkuma** [community/community#38081] [Other Feature Feedback, Questions, & Ideas]
We have 150+ repositories repositories for micro APIs and we have created multiple workflows in each APIs, Hence we need to create a dashboard where we can get the all branches activities and deployment (CICD) in one page. 
Do we have any tool available which we can integrate with GitHub actions and create the dashboard. something like DevOps KPI.

**Comment by ossanna16** [comment]
Hi there @conrkuma and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
